src/PresentationalComponents/Charts/ActionsOverviewDonut.js

```diff
@@ -10,7 +10,7 @@ class AdvisorOverviewDonut extends React.Component {
     getChart = (theme, typeNames) => <ChartDonut
         data={ this.props.category.map((value, key) => ({ x: typeNames[key], y: value, label: `${typeNames[key]}: ${value}` })) }
         theme={ theme }
-        height={ 200 }
+        height={ 200 }ChartTheme
```

Comment:
eekkkkk 😭

Reply:
I really hate the default 'middle click paste' option on mouse configurations...